pkg/proxy/net/packetio_mysql.go

```diff
@@ -74,7 +73,8 @@ func (p *PacketIO) WriteInitialHandshake(capability Capability, salt []byte, aut
 func (p *PacketIO) WriteSwitchRequest(authPlugin string, salt []byte) error {
 	length := 1 + len(authPlugin) + 1 + len(salt) + 1
 	data := make([]byte, 0, length)
-	data = append(data, mysql.AuthSwitchRequest)
+	// check https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_connection_phase_packets_protocol_auth_switch_request.html
+	data = append(data, 0xfe)
```

Comment:
Why not also define this constant?

Reply:
Defined.

Because it is duplicated with EOF header byte. Also, it is only used here once.